mongo/integration/sdam_error_handling_test.go

```diff
@@ -33,8 +33,8 @@ func TestSDAMErrorHandling(t *testing.T) {
 	}
 	baseMtOpts := func() *mtest.Options {
 		mtOpts := mtest.NewOptions().
-			Topologies(mtest.ReplicaSet). // Don't run on sharded clusters to avoid complexity of sharded failpoints.
-			MinServerVersion("4.0").      // 4.0+ is required to use failpoints on replica sets.
+			Topologies(mtest.ReplicaSet, mtest.Single). // Don't run on sharded clusters to avoid complexity of sharded failpoints.
```

Comment:
This was mostly to facilitate local testing so I didn't need to set up a replica set. I kept it because it may make running the tests locally easier for other people, too (and they should run fine for the standalone deployment on Evergreen).